### PR TITLE
Specify default values

### LINKED ISSUE #46: Update features description
https://github.com/tolstoyevsky/hubot-happy-birthder/blame/master/README.md#L11
Actually, the script does NOT use the `simpsons-birthday` search query, it uses the value defined in `TENOR_SEARCH_TERM` or default, which is `false`.

### PR DESCRIPTION
**Pull request by eugulixes**
Fixes #46.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,15 +16,15 @@ The script requires [hubot-auth](https://github.com/hubot-scripts/hubot-auth).
...
-| `BIRTHDAY_CRON_STRING`              | Allows specifying the frequency with which the script writes birthday messages to users. The value of this parameter must follow the [Cron Format](https://github.com/node-schedule/node-schedule#cron-style-scheduling). |
-| `BIRTHDAY_ANNOUNCEMENT_BEFORE_CNT`  | Sets how long before the event occurs the reminder will be triggered. |
-| `BIRTHDAY_ANNOUNCEMENT_BEFORE_MODE` | Unit of time. The possible values are (the corresponding shorthands are specified in the brackets): `years` (`y`), `quarters` (`Q`), `months` (`M`), `weeks` (`w`), `days` (`d`), `hours` (`h`), `minutes` (`m`), `seconds` (`s`), `milliseconds` (`ms`). |
+| Parameter                           | Description | Default |
+|-------------------------------------|-------------|---------|
+| `TENOR_API_KEY`                     | Сlient key for privileged API access. This is the only **mandatory** parameter. | |
+| `TENOR_IMG_LIMIT`                   | Fetches up to the specified number of result, but not more than **50**. By default the value of the variable and the corresponding API parameter is **20**. | 50 |
```
Comment (sibiryoff) [inline README.md]: 'Description' column: ``` By default the value of the variable and the corresponding API parameter is **20** ```
'Default' column: ``` 50 ```

**Review by sibiryoff — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (eugulixes) [reply README.md]: It's tricky. **20** is the default value in Tenor API.

Comment (sibiryoff) [reply README.md]: Ok, but we have following words here: ```by default the value of the variable ... is **20**```, that doesn't match the value in 'default' column and is confusing.

Comment (eugulixes) [reply README.md]: Never thought about it before=) I'll fix it.

Comment (eugulixes) [reply README.md]: Fixed.

**Comment by sibiryoff** [comment]
The script launches and birthdays list works, but how can I check other functions? Birthday message doesn't work with this settings, even if I set somebody's birthday for today.

**Comment by eugulixes** [comment]
@ukhryab, probably the reason is that the `0 0 7 * * *` schedule is used by default?=)

**Comment by sibiryoff** [comment]
@eugulixes yes, and I wonder how can I test it quickly since `0 0 7 * * *` stands for `7:00` ))

**Comment by eugulixes** [comment]
@ukhryab, I've just done it by simply commiting and pushing `15 * * * * *`.

**Review by sibiryoff — DISMISSED** [review]

**Review by sibiryoff — APPROVED** [review]

**Pull request merged**